test/jdk/javax/crypto/Cipher/TestEmptyModePadding.java

```diff
@@ -24,33 +24,63 @@
 
 /*
  * @test
- * @bug 8358159
- * @summary test that the Cipher.getInstance() handles
- * transformations with empty mode and/or padding
- * @run main TestEmptyModePadding
+ * @bug 8358159 8359388
```

Comment:
I think we should remove 8358159 from the bug line, as this is clearly not testing that bugfix anymore.

Reply:
Yes, good point.